salt/pillar/postgres.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+'''
+Retrieve Pillar data by doing a postgres query
+
+:maturity: new
+:depends: psycopg2
+:platform: all
+
+Complete example
+=====================================
+
+.. code-block:: yaml
+
+    postgres:
+      user: 'salt'
+      pass: 'super_secret_password'
+      db: 'salt_db'
```

Comment:
:+1: to this, would be much better to define a connection_url, in case something gets added in the future, we don't have to update the configuration options that salt accepts, it could just be available immediately by using the connection url.